language_formatters_pre_commit_hooks/pretty_format_java.py

```diff
@@ -70,6 +70,12 @@ def pretty_format_java(argv: typing.Optional[typing.List[str]] = None) -> int:
         dest="aosp",
         help="Formats Java code into AOSP format",
     )
+    parser.add_argument(
+        "--google-java-formatter-jar",
+        dest="google_java_formatter_jar",
+        default=None,
+        help="Path to Google Java Formatter jar file. Will be downloaded in case of None (default: %(default)).",
```

Comment:
Please:
* make explicit the the fact that version will be ignored in such case
* replace "if None" with "if not defined"
* [optional] leverage argparse mutually exclusive arguments to prevent version and path to be defined together

Reply:
Will be working on it 🚀